.circleci/cimodel/data/caffe2_build_definitions.py

```diff
@@ -30,6 +30,7 @@ class Conf:
     def get_cudnn_insertion(self):
 
         omit = self.language == "onnx_py2" \
```

Comment:
this (`onnx_py2`) can be removed now?

Reply:
I would suggest to keep py2 one until we retire py2 internally. py2 helped us caught several bugs which were not manifested in py3.